src/Language/Haskell/Refact/Utils/ExactPrint.hs

```diff
@@ -160,3 +177,146 @@ balanceAllComments la
       return t
 
 -- ---------------------------------------------------------------------
+--Useful helper function that logs the current refact parsed
+
+exactPrintParsed :: RefactGhc ()
+exactPrintParsed = do
```

Comment:
There are various `logXXX` functions in `MonadFunctions`, including `logExactPrint`.   Consider moving this and the next one there too.